contrib/localnet/orchestrator/smoketest/cmd/smoketest/stress.go

```diff
@@ -1,321 +1,377 @@
+
+	// -------------- TEST BEGINS ------------------
+
+	fmt.Println("**** STRESS TEST BEGINS ****")
+	fmt.Println("	1. Periodically Withdraw ETH from ZEVM to EVM - goerli")
+	fmt.Println("	2. Display Network metrics to monitor performance [Num Pending outbound tx], [Num Trackers]")
+
+	smokeTest.WG.Add(2)
+	go WithdrawCCtx(smokeTest)       // Withdraw USDT from ZEVM to EVM - goerli
+	go EchoNetworkMetrics(smokeTest) // Display Network metrics periodically to monitor performance
+
+	smokeTest.WG.Wait()
+}
+
+// WithdrawCCtx withdraw USDT from ZEVM to EVM
+func WithdrawCCtx(sm *runner.SmokeTestRunner) {
+	ticker := time.NewTicker(time.Millisecond * time.Duration(stressTestArgs.txnInterval))
+	for {
+		select {
+		case <-ticker.C:
+			WithdrawETHZRC20(sm)
+		}
+	}
+}
+
+func EchoNetworkMetrics(sm *runner.SmokeTestRunner) {
+	ticker := time.NewTicker(time.Second * StatInterval)
+	var queue = make([]uint64, 0)
+	var numTicks = 0
+	var totalMinedTxns = uint64(0)
+	var previousMinedTxns = uint64(0)
+	chainID, err := getChainID(sm.GoerliClient)
+
+	if err != nil {
+		panic(err)
+	}
+
+	for {
+		select {
+		case <-ticker.C:
+			numTicks++
+			// Get all pending outbound transactions
+			cctxResp, err := sm.CctxClient.CctxListPending(context.Background(), &crosschaintypes.QueryListCctxPendingRequest{
+				ChainId: chainID.Int64(),
+			})
+			if err != nil {
+				continue
+			}
+			sends := cctxResp.CrossChainTx
+			sort.Slice(sends, func(i, j int) bool {
+				return sends[i].GetCurrentOutTxParam().OutboundTxTssNonce < sends[j].GetCurrentOutTxParam().OutboundTxTssNonce
+			})
+			if len(sends) > 0 {
+				fmt.Printf("pending nonces %d to %d\n", sends[0].GetCurrentOutTxParam().OutboundTxTssNonce, sends[len(sends)-1].GetCurrentOutTxParam().OutboundTxTssNonce)
+			} else {
+				continue
+			}
+			//
+			// Get all trackers
+			trackerResp, err := sm.CctxClient.OutTxTrackerAll(context.Background(), &crosschaintypes.QueryAllOutTxTrackerRequest{})
+			if err != nil {
+				continue
+			}
+
+			currentMinedTxns := sends[0].GetCurrentOutTxParam().OutboundTxTssNonce
+			newMinedTxCnt := currentMinedTxns - previousMinedTxns
+			previousMinedTxns = currentMinedTxns
+
+			// Add new mined txn count to queue and remove the oldest entry
+			queue = append(queue, newMinedTxCnt)
+			if numTicks > 60/StatInterval {
+				totalMinedTxns -= queue[0]
+				queue = queue[1:]
+				numTicks = 60/StatInterval + 1 //prevent overflow
+			}
+
+			//Calculate rate -> tx/min
+			totalMinedTxns += queue[len(queue)-1]
+			rate := totalMinedTxns
+
+			numPending := len(cctxResp.CrossChainTx)
+			numTrackers := len(trackerResp.OutTxTracker)
+
+			fmt.Println("Network Stat => Num of Pending cctx: ", numPending, "Num active trackers: ", numTrackers, "Tx Rate: ", rate, " tx/min")
+		}
+	}
+}
+
+func WithdrawETHZRC20(sm *runner.SmokeTestRunner) {
+	defer func() {
+		zevmNonce.Add(zevmNonce, big.NewInt(1))
+	}()
+
+	ethZRC20 := sm.ETHZRC20
+
+	sm.ZevmAuth.Nonce = zevmNonce
```

Comment:
I'm using the nonce so that I can send multiple txns in the same block. I was getting an issue otherwise, the sequence was out of order on the sdk side so I had to keep track of the nonce myself.